activities/config_johnpolo.json

```diff
@@ -0,0 +1,16 @@
+{
+  "tasks": [
+      {
+      "layers": [
+        [
+          "d.rast", "map=elev_lid792_1m"
```

Comment:
You probably want `feat_verticalcurv` here. Since it will be hardcoded in the `run_topo_par` function, it will exist when Tangible Landscape runs this function.

Then you will see this in the artifact from Render activities check.

Reply:
I substituted feat_verticalcurv for elev...

Reply:
Great. I checked the result in the artifact and it looks good!